src/jit/morph.cpp

```diff
@@ -12576,8 +12577,10 @@ GenTreePtr Compiler::fgMorphSmpOp(GenTreePtr tree, MorphAddrContext* mac)
 
                     return tree;
                 }
+#endif // LEGACY_BACKEND
                 break;
 
+#ifdef LEGACY_BACKEND
             case GT_QMARK:
```

Comment:
Those are some morph attempts that don't seem to match any of the patterns generated by the importer. See the updated PR description.

No diffs.